imagepipeline/src/main/java/com/facebook/imagepipeline/producers/LocalFileFetchProducer.java

```diff
@@ -33,9 +34,9 @@ public LocalFileFetchProducer(
 
   @Override
   protected EncodedImage getEncodedImage(final ImageRequest imageRequest) throws IOException {
-    return getEncodedImage(
-        new FileInputStream(imageRequest.getSourceFile().toString()),
-        (int) imageRequest.getSourceFile().length());
+      File sourceFile = imageRequest.getSourceFile();
+      return getEncodedImage(sourceFile,
+              new FileInputStream(imageRequest.getSourceFile().getPath()), (int) sourceFile.length());
```

Comment:
It's just a coding style, since you already have `sourceFile = imageRequest.getSourceFile()`, you can just do `sourceFile.getPath()` instead of `imageRequest.getSourceFile().getPath()`.